src/libstd/sys/unix/process/process_unix.rs

```diff
@@ -202,26 +202,26 @@ impl Command {
         // emscripten has no signal support.
         #[cfg(not(any(target_os = "emscripten")))]
         {
-            use crate::mem;
+            use crate::mem::{self, MaybeUninit};
             // Reset signal handling so the child process starts in a
             // standardized state. libstd ignores SIGPIPE, and signal-handling
             // libraries often set a mask. Child processes inherit ignored
             // signals and the signal mask from their parent, but most
             // UNIX programs do not reset these things on their own, so we
             // need to clean things up now to avoid confusing the program
             // we're about to run.
-            let mut set: libc::sigset_t = mem::uninitialized();
+            let mut set = MaybeUninit::<libc::sigset_t>::uninit();
             if cfg!(target_os = "android") {
                 // Implementing sigemptyset allow us to support older Android
                 // versions. See the comment about Android and sig* functions in
                 // process_common.rs
-                libc::memset(&mut set as *mut _ as *mut _,
+                libc::memset(set.as_mut_ptr() as *mut _,
```

Comment:
Why does this need to cast the raw ptr type around?
(I am extremely scared of raw ptr casts as they go wrong so easily.^^)

Reply:
If this just casts to `*mut u8` because it's a byte-wise `memset`, please spell out that type.

Reply:
It actually becomes a `*mut c_void`, but I can still make that explicit.

Reply:
Actually, I guess we can just use `ptr::write_bytes`.